erts/doc/src/erlang.xml

```diff
@@ -7441,26 +7441,18 @@ lists:map(
 	    anchor="statistics_scheduler_wall_time" since="OTP R15B01"/>
       <fsummary>Information about each schedulers work time.</fsummary>
       <desc>
-        <p>Returns a list of tuples with
-          <c>{<anno>SchedulerId</anno>, <anno>ActiveTime</anno>,
-          <anno>TotalTime</anno>}</c>, where
-          <c><anno>SchedulerId</anno></c> is an integer ID of the scheduler,
-          <c><anno>ActiveTime</anno></c> is
-          the duration the scheduler has been busy, and
-          <c><anno>TotalTime</anno></c> is the total time duration since
-          <seeerl marker="#system_flag_scheduler_wall_time">
-          <c>scheduler_wall_time</c></seeerl>
-          activation for the specific scheduler. Note that
-	  activation time can differ significantly between
-	  schedulers. Currently dirty schedulers are activated
-	  at system start while normal schedulers are activated
-	  some time after the <c>scheduler_wall_time</c>
-	  functionality is enabled. The time unit is undefined
-	  and can be subject to change between releases, OSs,
-	  and system restarts. <c>scheduler_wall_time</c> is only
-	  to be used to calculate relative values for scheduler
-	  utilization. <c><anno>ActiveTime</anno></c> can never
-	  exceed <c><anno>TotalTime</anno></c>.</p>
+        <p>Returns information describing how much time
+          <seeerl marker="#system_info_schedulers">normal</seeerl>
+          and <seeerl marker="#system_info_dirty_cpu_schedulers">dirty CPU</seeerl>
+          schedulers in the system have been busy. This value is normally a
+          better indicator of how much load an Erlang node is under instead of
+          looking at the CPU utilization privided by tools such as <c>top</c> or
+          <c>sysstat</c>. This is because <c>scheduler_wall_time</c> also includes
+          time where the scheduler is waiting for some other reasource (such as an internal
+          mutex) to be available but does not use the CPU. In order to better
```

Comment:
Could be worth noting that time spent by schedulers busy waiting for more work to become available is not included.

Reply:
> The definition of a busy scheduler is when it is not idle and **is not scheduling (selecting) a process or port**

I thought that the second part of this sentence was meant to convey that? I can change the wording on that to make it clearer.